lib/Importer/ProtobufLoader.cpp

```diff
@@ -37,34 +37,50 @@ Tensor *ProtobufLoader::getTensorByName(const std::string &name) {
   return tensors_[name];
 }
 
-Node *ProtobufLoader::getNodeByName(const std::string &name) {
+Node *ProtobufLoader::getNodeByName(const std::string &name,
+                                    bool allowNullptr) const {
   auto it = nodeByName_.find(name);
   if (it != nodeByName_.end()) {
     return it->second;
   }
 
-  llvm_unreachable("Could not find a node with this name.");
+  if (!allowNullptr) {
+    llvm_unreachable("Could not find a node with this name.");
```

Comment:
nit: I'd change this to assert, because now it's not "unreachable" by its nature.

Reply:
Done